fastlane/lib/fastlane/actions/git_branch.rb

```diff
@@ -26,8 +26,11 @@ def self.available_options
         []
       end
 
+      # rubocop:disable Lint/MissingKeysOnSharedArea
```

Comment:
Is there a purpose for disabling it here?

Reply:
With @milch suggestion, I was able to remove it 🙌